utils/content-negotiation.util.js

```diff
@@ -0,0 +1,50 @@
+const jsonToXmlConverter = require("js2xmlparser")
+const jsonToPlainText = require("json-to-plain-text");
+const jsonToHtml = require('json-to-html')
+
+
+const sendResponseToClient = (res,statusCode,resposeData)=>{
+    res.status(statusCode).send(resposeData);
+};
+
+
+const sendJsonResponse = (res,status,message,data)=>{
+    const responseData = {status: status, message:message, data:data};
+     sendResponseToClient(res,status,responseData);
+}
+
+const sendHtmlResponse = (res,status,data)=>{
+   const responseData = jsonToHtml(data);
+    sendResponseToClient(res,status,responseData);
+};
+
+const sendXmlResponse = (res,status,data)=>{
+    const responseData = jsonToXmlConverter.parse("data", data);
+    sendResponseToClient(res,status,responseData);
+};
+
+
+
+const sendTextResponse = (res,status,data)=>{
+    const responseData = jsonToPlainText.toPlainText(JSON.parse(JSON.stringify(data)));
+    sendResponseToClient(res,status,responseData);
+};
+
+const sendResponseInContentNegotiation = (req,res,status,data)=>{
```

Comment:
i think the whole functionality can be written in one single block, We will discuss about this